gradio/routes.py

```diff
@@ -141,8 +145,12 @@ def get_blocks(self) -> gradio.Blocks:
         return self.blocks
 
     @staticmethod
-    def create_app(blocks: gradio.Blocks) -> App:
-        app = App(default_response_class=ORJSONResponse)
+    def create_app(
+        blocks: gradio.Blocks, app_kwargs: Optional[Dict[str, Any]] = None
```

Comment:
For clarity and consistency with the rest of the codebase
```suggestion
        blocks: gradio.Blocks, app_kwargs: Dict[str, Any] | None = None
```